src/actions.ts

```diff
@@ -102,7 +105,7 @@ export async function executeActions(
           {
             GM_DBNAME: databaseName,
             GM_DBUSER: databaseUser,
```

Comment:
This is meant to be "the database user in GM_DBURL", but since GM_DBURL might be a root connection string, this should then be the root user? But that wouldn't necessarily be expected... I'm worried this'll be misused... Maybe when `actionSpec.root` is used `GM_DBUSER` should not be set? I'm not sure people use it anyway.

Let's not set it when `root`. Please also update the README to reflect this.

Reply:
Ooh, good catch! Updating